app.yaml

```diff
@@ -11,10 +11,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-runtime: custom
+runtime: ruby
 vm: true
 api_version: 1
 module: default
+# [START entrypoint]
+entrypoint: bundle exec foreman start --formation "$FORMATION"
+# [END entrypoint]
 
 resources:
```

Comment:
Check the "App configuration" section here: https://cloud.google.com/ruby/getting-started/hello-world